src/components/TeacherComponents/DownloadConfirmation/DownloadConfirmation.tsx

```diff
@@ -46,6 +57,15 @@ const DownloadConfirmation: FC<DownloadConfirmationProps> = ({
     lessonSlug,
     unitSlug: unitSlug ?? undefined,
     programmeSlug: programmeSlug ?? undefined,
+    source: isCanonical ? "download-canonical" : "download-browse",
+    curriculumTrackingProps: {
+      lessonName: lessonTitle,
+      unitName: unitTitle ?? "",
+      keyStageSlug,
+      keyStageTitle,
+      subjectSlug,
+      subjectTitle,
+    },
   });
   console.log(shareIdRef, shareIdKeyRef);
```

Comment:
are you meaning to console log this?

Reply:
sorry. yes for now as we need to use the variables to make the linter happy